### PR TITLE
Fix neovim deps

### PR DESCRIPTION
**Pull request by Tranquility**
The neovim [wiki](https://github.com/neovim/neovim/wiki/Installing#gentoo-linux) lists this overlay in the installing guide. So I thought I should fix the ebuild.

### CODE REVIEW

File: app-editors/neovim/neovim-9999.ebuild [hunk 1]
```diff
@@ -4,16 +4,10 @@
...
+inherit cmake-utils git-r3
 
-if [ "${PV}" = "9999" ]; then
-	EGIT_REPO_URI="git://github.com/neovim/neovim.git"
-	inherit git-r3
-	KEYWORDS=""
-else
-	SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64"
-fi
```
Comment (junghans) [inline app-editors/neovim/neovim-9999.ebuild]: Please leave this here, it makes it easier to split out ebuilds for releases!

File: app-editors/neovim/ChangeLog [hunk 2]
```diff
@@ -2,6 +2,9 @@
 # Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
 # $Header: $
 
+  17 Feb 2015;  <mail@ole-reifschneider.de> neovim-9999.ebuild:
```
Comment (junghans) [inline app-editors/neovim/ChangeLog]: Please put your name in here, too. You can use:
```
export ECHANGELOG_USER="Frist Last <mail@ole-reifschneider.de>"
```

File: dev-libs/libtermkey/ChangeLog [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# ChangeLog for dev-libs/libtermkey
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+*libtermkey-9999 (17 Feb 2015)
+
+  17 Feb 2015;  <mail@ole-reifschneider.de> +libtermkey-9999.ebuild, +metadata.xml:
```
Comment (junghans) [inline dev-libs/libtermkey/ChangeLog]: Same here, missing name!

File: dev-libs/libtermkey/libtermkey-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+HOMEPAGE="http://www.leonerd.org.uk/code/libtermkey/"
+EBZR_REPO_URI="http://bazaar.leonerd.org.uk/c/libtermkey/"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
```
Comment (junghans) [inline dev-libs/libtermkey/libtermkey-9999.ebuild]: Drop `DEPEND` and `RDEPEND` if they are empty.

File: dev-libs/libtermkey/libtermkey-9999.ebuild [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+src_install() {
+	emake PREFIX="/usr" DESTDIR="${D}" install
```
Comment (junghans) [inline dev-libs/libtermkey/libtermkey-9999.ebuild]: use PREFIX="${EPREFIX}/usr" here.

File: dev-libs/unibilium/unibilium-9999.ebuild [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+DESCRIPTION="A terminfo parsing library"
+HOMEPAGE="https://github.com/mauke/unibilium"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
```
Comment (junghans) [inline dev-libs/unibilium/unibilium-9999.ebuild]: Same here, drop!

File: dev-libs/unibilium/unibilium-9999.ebuild [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+src_install() {
+	emake PREFIX="/usr" DESTDIR="${D}" install
```
Comment (junghans) [inline dev-libs/unibilium/unibilium-9999.ebuild]: Same here, add `${EPREFIX}`.

**Comment by junghans** [comment]
If possible, next time please do one pull request per package.

**Comment by Tranquility** [comment]
> If possible, next time please do one pull request per package.

I thought it's easier to have one because otherwise one would depend on the other two.

**Comment by Tranquility** [comment]
I updated it.

**Comment by junghans** [comment]
Looks very good and ready to merge, expect this repoman warning from travis:
```
 dependency.badmasked
   app-editors/neovim/neovim-9999.ebuild: DEPEND: **(empty profile) ['=dev-libs/msgpack-9999', 'dev-lua/messagepack']
```
Could you ahead and add `dev-libs/msgpack` and `dev-lua/messagepack` to this or another pull request!

**Comment by jlec** [comment]
@Tranquility please use repoman or repo-commit to commit, not plain git commit.

**Comment by Tranquility** [comment]
I added the missing ebuilds and recommited with repoman.

**Comment by junghans** [comment]
Two more things:
```
  dependency.missingslot
   dev-lua/messagepack/messagepack-9999.ebuild: RDEPEND: '>=dev-lang/lua-5.1' matches more than one slot, please specify an explicit slot and/or use the := or :* slot operator
  dependency.unknown
   app-editors/neovim/neovim-9999.ebuild: DEPEND: dev-lua/lulpeg[lpeg_replace]
```

I think `messagpack` needs `lua-5.1:*` and `dev-lua/lulpeg` needs to be added, too.

**Comment by Tranquility** [comment]
@junghans done

**Comment by Tranquility** [comment]
thanks!

**Pull request merged**